third_party/mini_chromium/CMakeLists.txt

```diff
@@ -151,3 +163,15 @@ if(WIN32)
         )
     endif()
 endif()
+
+add_library(crashpad::mini_chromium ALIAS mini_chromium)
+
+crashpad_install_target(mini_chromium)
+
+if(CRASHPAD_ENABLE_INSTALL_DEV)
+    file(GLOB_RECURSE MINICHROMIUM_HEADERS "*.h")
```

Comment:
Yeah, cmake discourages glob's for building because it will not detect a new file added to a target.
But I'll change all install's to `install(DIRECTORY)`.